sdk/storage/ci.yml

```diff
@@ -13,18 +26,29 @@ pr:
   branches:
     include:
     - master
+    - feature/*
+    - hotfix/*
+    - release/*
   paths:
     include:
     - sdk/storage/
 
-resources:
-  repositories:
-    - repository: azure-sdk-tools
-      type: github
-      name: Azure/azure-sdk-tools
-      endpoint: azure
-
-jobs:
-- template: ../../eng/pipelines/templates/jobs/archetype-sdk-client.yml
+stages:
+- template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
     ServiceDirectory: storage
+    ArtifactName: packages
+    Artifacts:
+    - name: Azure.Storage.Blobs
+      safeName: AzureStorageBlobs
+    - name: Azure.Storage.Blobs.Batching
+      safeName: AzureStorageBlobsBatching
```

Comment:
You might have to rebase after this PR comes out since we renamed Batching to Batch (PR #8350 )

Reply:
Rebased. Thanks for the heads up.